pkg/storage/bucket/s3/bucket_client.go

```diff
@@ -39,13 +44,14 @@ func newS3Config(cfg Config) (s3.Config, error) {
 	}
 
 	return s3.Config{
-		Bucket:    cfg.BucketName,
-		Endpoint:  cfg.Endpoint,
-		Region:    cfg.Region,
-		AccessKey: cfg.AccessKeyID,
-		SecretKey: cfg.SecretAccessKey.String(),
-		Insecure:  cfg.Insecure,
-		SSEConfig: sseCfg,
+		Bucket:          cfg.BucketName,
+		Endpoint:        cfg.Endpoint,
+		Region:          cfg.Region,
+		AccessKey:       cfg.AccessKeyID,
+		SecretKey:       cfg.SecretAccessKey.String(),
+		Insecure:        cfg.Insecure,
+		PutUserMetadata: map[string]string{awsStorageClassHeader: cfg.StorageClass},
```

Comment:
Might be being overly cautious here: it's probably safest for the default value of `cfg.StorageClass` to be an empty string, and to not set `PutUserMetadata` at all if `StorageClass` is not set - some S3-compatible storage services may not support `STANDARD`.

Reply:
Hi! thank you so much for the review!
It sounds like a very good point - Ive tried to edit so the default value of the storage class in the config.go will be an empty string, but it seemed to fail some tests because the object storage class "" doesn't exist.
Maybe I should edit it in this file to be an empty string?
I fixed the pr according to all the comments, but I still think how to fix so solve this one.

Reply:
And also I'm thinking - maybe its better if it will have an actual default value that will be supported by default (If it exists?)

Reply:
Rather than setting it to an empty string, I would simply not set it at all (which is what we were doing before).